Sources/SnapshotTesting/AssertSnapshot.swift

```diff
@@ -323,9 +331,11 @@ public func verifySnapshot<Value, Format>(
       #endif
     }
 
+    let diffToolMessage = diffToolBuilder?(snapshotFileUrl, failedSnapshotFileUrl) ??
+      (diffTool
+        .map { "\($0) \"\(snapshotFileUrl.path)\" \"\(failedSnapshotFileUrl.path)\"" })
     let diffMessage =
-      diffTool
```

Comment:
As it stands this PR will break existing use of `diffTool`. We're down to support this feature (as an alternative to #508), but can you make the following change?

1. Update `var diffTool` to be a computed getter/setter to `diffToolBuilder`
2. Consider some alternative names? Maybe `diffToolCommand`?

Reply:
The existing `diffTool` always had precedence over the closure, but agreed a custom getter+setter makes the diffMessage site cleaner. However it wasn't obvious to me how to construct an empty URL for the `diffTool` getter so I changed the `diffToolCommand` closure to receive the path as String rather than the file URL.

Renamed it to `diffToolCommand` instead.